src/components/About/about.css

```diff
@@ -0,0 +1,59 @@
+.about-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.about-text {
+  margin-left: 15.1vw;
```

Comment:
Although not wrong, using vw and vh as margin can lead to undesirable results sometimes when screen becomes very big or very small, be careful.

Reply:
Because it is always relative to viewport size and not container

Reply:
Should I use px instead?

Reply:
Depends on the use case but I usually use px when positioning inside fixed sized containers